tools/dissector/sample_manager.cpp

```diff
@@ -154,12 +154,13 @@ namespace OpenDDS
         }
       }
 
-      void visit (itl::Fixed&) {
-        // TODO Fixed types are supported in OpenDDS now, and the dissector
-        // should handle them.
+      void visit (itl::Fixed& fixed) {
+#ifndef ACE_HAS_CDR_FIXED
```

Comment:
The preprocessor logic looks backwards to me: If ACE_HAS_CDR_FIXED is not defined, attempt to use Fixed, otherwise warn.